docs/start.md

```diff
@@ -44,3 +44,10 @@ There are two main ways of running quantum algorithms using Cirq:
     - [Google Quantum Computing Service](tutorials/google/start.ipynb)
     - [Alpine Quantum Technologies](tutorials/aqt/getting_started.ipynb)
     - [Pasqal](tutorials/pasqal/getting_started.ipynb) 
+
+# Next steps
+
+- Learn about the [Basics](tutorials/basics.ipynb) of Cirq, like the Circuit, Qubit and Simulator classes and how to use them
+- Explore all the numerous [Tutorials and examples](tutorials/)
+- Check out the [API docs](https://quantumai.google/reference/python/cirq)
+- Learn more about the Cirq concepts in-depth in our guides 
```

Comment:
nit: should there be periods on the end of the sentences above?

Reply:
Done!